tests/CommandTest.php

```diff
@@ -51,22 +51,6 @@ public function testAlterColumn(): void
         $this->assertSame('ntext', $columns['email']->getDbType());
     }
 
-    /**
```

Comment:
After remove will be using main tests for batchInsert?

Reply:
yes.